cfme/tests/infrastructure/test_quota.py

```diff
@@ -40,7 +40,14 @@ def template_name(provider):
     if provider.one_of(RHEVMProvider):
         return provider.data.templates.get('full_template')['name']
     elif provider.one_of(VMwareProvider):
-        return provider.data.templates.get('big_template')['name']
+        if provider.name == "vSphere 6 (nested)":
```

Comment:
We should never trigger things in integration_tests based on specific keys from our yaml files.

This should be fixed by updating our vmware yaml dictionaries to include a 'big_template' definition, using rhel74_template.

Nowhere in the code should it be looking for Fedora_24WS template either, since that's just the specific name of a template on one of our current providers.

The lookup of values from yaml in test fixtures and framework components should be abstracted and structured to produce a reasonable outcome when the yaml doesn't contain the requisite value.

For example, here when the yaml for a certain provider type doesn't include a big_template key, it would be optimal to catch that exception and call pytest.skip(), so that the test is collected and counted in results, but as a skip (failure) because the necessary resource was missing. Taking this to the extreme, if we had no providers to test against, and tried to collect and run all tests in the repo, we would have a very low pass rate with  almost all failures being skipped tests.

Beyond making the approach to the lookup of these templates abstract and improving this fixture by calling pytest.skip, I think the problem you are trying to solve should be fixed by updating cfme_data.yaml in the appropriate RH repo.

Reply:
Yes @mshriver as you said, we need to update ```cfme_data.yaml```. But before updating this file with other template as ```big_template```. I requested @kedark3 for adding **Fedora_24WS** template to vSphere6-nested provider. Which will solve the problem of missing template ```Fedora_24WS``` and also no need to update cfme_data.yaml.
Thank you.

Reply:
```Fedora_24WS``` template is added for vSphere6-nested provider. Hence closing this PR.